include/cpp_lmdb/transactions.hpp

```diff
@@ -124,13 +124,11 @@ class transaction {
         requires(
             details::key_value_trait_helper<KeyValueTrait>::duplicates_enabled)
     {
-        auto cursor = make_cursor(_api, _txn.get(), _db_index);
+        auto cursor = details::make_cursor(_api, _txn.get(), _db_index);
```

Comment:
If you are interested, integration test for `iterate_by_key` is missing as well (#25) :)